### PR TITLE
feat(pr: issue-346): add client for smalltalk app (reactjs) in auth

### PR DESCRIPTION
**Pull request by live-dev999**

### CODE REVIEW

File: src/Services/auth/O2NextGen.Auth.Web/Utilities/Base64QrCodeGenerator.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+using SkiaSharp;
+using SkiaSharp.QrCode;
+
+namespace O2NextGen.Auth.Web.Utilities
+{
+    public class Base64QrCodeGenerator : IBase64QrCodeGenerator
+    {
+        public string Generate(Uri target)
+        {
+            // https://github.com/guitarrapc/SkiaSharp.QrCode
```
Comment (AlbusaOxyuranus) [inline src/Services/auth/O2NextGen.Auth.Web/Utilities/Base64QrCodeGenerator.cs:11]: Maybe you can delete the comments?

File: src/Services/auth/O2NextGen.Auth.Web/StartupHelpers/DatabaseExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    {
+        internal static async Task EnsureDbUpToDateAsync(this IWebHost host)
+        {
+            using (var scope = host.Services.CreateScope())
+            {
+                var hostingEnvironment = scope.ServiceProvider.GetRequiredService<IHostingEnvironment>();
+                var authDbContext = scope.ServiceProvider.GetRequiredService<AuthDbContext>();
+                await authDbContext.Database.MigrateAsync();
+
+                // var grantDbContext = scope.ServiceProvider.GetRequiredService<PersistedGrantDbContext>();
```
Comment (AlbusaOxyuranus) [inline src/Services/auth/O2NextGen.Auth.Web/StartupHelpers/DatabaseExtensions.cs:19]: I think, you need delete it

File: src/Services/auth/O2NextGen.Auth.Web/Utilities/Base64QrCodeGenerator.cs (hunk 1 above)
Comment (live-dev999) [reply src/Services/auth/O2NextGen.Auth.Web/Utilities/Base64QrCodeGenerator.cs:11]: Sure. It's good idea...

File: src/Services/auth/O2NextGen.Auth.Web/StartupHelpers/DatabaseExtensions.cs (hunk 2 above)
Comment (AlbusaOxyuranus) [reply src/Services/auth/O2NextGen.Auth.Web/StartupHelpers/DatabaseExtensions.cs:19]: I think it will do later

**Pull request merged**